### PR TITLE
update CH-benCHmark

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: comments in https://github.com/pingcap/docs-cn/pull/10762
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by dbsid — APPROVED** [review]

File: benchmark/benchmark-tidb-using-ch.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ creating view revenue1
 部署 TiFlash 后，TiFlash 并不会自动同步 TiKV 数据，你需要执行以下 SQL 语句创建整库的 TiFlash 副本。创建 TiFlash 副本后，系统自动实时同步最新数据到 TiFlash 组件。以下例子中，集群中部署了两个 TiFlash 节点，将 replica 设置为 2。
 
 ```
-alter database tpcc set tiflash replica 2;
+ALTER DATABASE tpcc SET tiflash replica 2;
```
Comment (hfxsd) [inline benchmark/benchmark-tidb-using-ch.md:72]: TIFLASH REPLICA 好像也应该用大写，这篇文档有示例 https://docs.pingcap.com/tidbcloud/create-tiflash-replicas#create-tiflash-replicas-for-databases
```suggestion
ALTER DATABASE tpcc SET TIFLASH REPLICA 2;
```

File: benchmark/benchmark-tidb-using-ch.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ creating view revenue1
 部署 TiFlash 后，TiFlash 并不会自动同步 TiKV 数据，你需要执行以下 SQL 语句创建整库的 TiFlash 副本。创建 TiFlash 副本后，系统自动实时同步最新数据到 TiFlash 组件。以下例子中，集群中部署了两个 TiFlash 节点，将 replica 设置为 2。
```
Comment (hfxsd) [inline benchmark/benchmark-tidb-using-ch.md:69]:
```suggestion
部署 TiFlash 后，TiFlash 并不会自动同步 TiKV 数据，你需要执行以下 SQL 语句创建整库的 TiFlash 副本。创建 TiFlash 副本后，系统自动实时同步最新数据到 TiFlash 组件。以下例子中，将 replica 设置为 2，会在集群中部署两个 TiFlash 节点。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: benchmark/benchmark-tidb-using-ch.md (hunk 2 above)
Comment (qiancai) [reply benchmark/benchmark-tidb-using-ch.md:69]: “将 replica 设置为 2，会在集群中部署两个 TiFlash 节点。”可能不太准确，需要先部署了两个 TiFlash 节点，这次才能设置为 2。 @dbsid 可以请确认下吗？

Comment (dbsid) [reply benchmark/benchmark-tidb-using-ch.md:69]: 是的，需要先部署两个 TiFlash 节点。

Comment (qiancai) [inline benchmark/benchmark-tidb-using-ch.md:69]:
```suggestion
部署 TiFlash 后，TiFlash 并不会自动同步 TiKV 数据，你需要执行以下 SQL 语句创建整库的 TiFlash 副本。创建 TiFlash 副本后，系统自动实时同步最新数据到 TiFlash 组件。例如，当集群中部署了两个 TiFlash 节点时，如果将 replica 设置为 2，执行以下 SQL 语句将创建 2 个 TiFlash 副本。
```

**Pull request merged**